content/kit/wiring.md

```diff
@@ -1,3 +1,14 @@
 ---
 title: Wiring
 ---
+
+Ensuring your robot is wired correctly is incredibly important. If wired incorrectly, your kit may not work correctly, or may be damaged.
+
+## Wire Colouring
+Ground should be Black, and nothing else should be black. This is in the interest of both safety and organisation. Being able to see where all your ground cables allows you to easily track cable runs and diagnose electronic issues. It also greatly reduces the chance of running an actual current down the ground, which will serious damage the kit!
+
+## Keep your solders clean
+Sometimes, you'll need to build your own circuits to extend out kit, which likely requires soldering. Keep an eye on your solders so they don't create a bridge between components. This will likely lead to the electronics not working correctly.
```

Comment:
**This will likely lead to the electronics not working correctly.** You could add if you inadvertently bridge a VCC/GND connection it could lead to sad times for everyone involved.